lib/ssm_config/migration_helper.rb

```diff
@@ -0,0 +1,61 @@
+module SsmConfig
+  class MigrationHelper
+    def initialize(file_name)
+      @file_name = file_name
+      @model_name = SsmConfig::SsmStorage::Db::ACTIVE_RECORD_MODEL
+    end
+
+    def migrate
+      added = []
+      keys_hash = accessor_key_hash(hash) # starting layer is always hash
+      last = nil
+      keys_hash.each do |accessor_key, value|
+        last = accessor_key
+        added.push(@model_name.constantize.create!(:file => @file_name, :accessor_keys => accessor_key, :value => value.to_s, :datatype => determine_class(value)))
+      end
+    rescue ActiveRecord::RecordInvalid => e
+      Rails.logger.error("#{e.message} was raised because of faulty data with accessor_key #{last}")
```

Comment:
Add a spec for this. You can probably to it with `allow(model_name).to receive(create!).and raise` or something like that.

Reply:
Got it. I wasn't able to figure it out earlier and reasoned its more of an ActiveRecord problem, but I'll look into it more